### PR TITLE
CSS Adaptive :D

### PR DESCRIPTION
**Pull request by Melinx**
There you go. Yah!

### CODE REVIEW

File: staff/Melinx/css-adaptive/css/mobile.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+}
+nav, aside {
+    background-color: greenyellow;
+    height: 20%;
+    align-content: center;
+
+
+}
+section {
+    background-color: pink;
```
Comment (agandia9) [inline staff/Melinx/css-adaptive/css/mobile.css]: [Kawaiiiiii Pink](https://i.imgflip.com/wyu3x.jpg)

File: staff/Melinx/css/index.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+            height:100%;
+            float: right;
+        }
+
+        footer {
+            background-color: burlywood
+
+        }
+
+    </style>
```
Comment (agandia9) [inline staff/Melinx/css/index.html]: use both style and link...?

File: staff/Melinx/css-adaptive/css/desktop.css [hunk 3]
```diff
@@ -0,0 +1,37 @@
+
+html, body {
```
Comment (manuelbarzi) [inline staff/Melinx/css-adaptive/css/desktop.css]: TIP move common styles (mobile / desktop) to main.css

File: staff/Melinx/css-adaptive/css/desktop.css [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    height: 100%;
+    width: 100%;
```
Comment (manuelbarzi) [inline staff/Melinx/css-adaptive/css/desktop.css]: TIP is ```width: 100%``` required?

File: staff/Melinx/css-adaptive/index.html [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    <title>Document</title>
+    <link rel="stylesheet" href="css/main.css">
+   
+</head>
+
+<body>
+
+    <header>header</header>
+    <main>
+        <nav class="col">nav</nav>
```
Comment (manuelbarzi) [inline staff/Melinx/css-adaptive/index.html]: WHAT do you need selector ```col```for? AFAIS not used in styles...

File: staff/Melinx/css/index.html [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+            font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif
+        }
+
+        header {
+            background-color: burlywood
+        }
+
+        main {
+            background-color: orangered;
+            display:inline-block ;
```
Comment (manuelbarzi) [inline staff/Melinx/css/index.html]: TIP is ```display: inline-block```required?

File: staff/Melinx/css/index.html [hunk 7]
```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <style>
+        h1 {
```
Comment (manuelbarzi) [inline staff/Melinx/css/index.html]: MOVE styles to external file (main.css)

File: staff/Melinx/css-adaptive/css/mobile.css (hunk 1 above)
Comment (Melinx) [reply staff/Melinx/css-adaptive/css/mobile.css]: fancy a piña colada?

File: staff/Melinx/css-adaptive/css/desktop.css (hunk 3 above)
Comment (Melinx) [reply staff/Melinx/css-adaptive/css/desktop.css]: done, thx!

File: staff/Melinx/css-adaptive/css/desktop.css (hunk 4 above)
Comment (Melinx) [reply staff/Melinx/css-adaptive/css/desktop.css]: removed width 100%

File: staff/Melinx/css-adaptive/index.html (hunk 5 above)
Comment (Melinx) [reply staff/Melinx/css-adaptive/index.html]: true that. Removed class "col"

File: staff/Melinx/portfolio/portfolio.html [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles/main.css">
+    <link rel="icon" type="image/png" href="images/favicon-palm.png">
+    <link href="https://fonts.googleapis.com/css?family=Anonymous+Pro|Cairo|News+Cycle" rel="stylesheet">
+
+    <title>Portfolio</title>
```
Comment (EzequielBurgosL) [inline staff/Melinx/portfolio/portfolio.html:12]: You've merged and pushed this project successfully! 💃  Nuttin' to wurry bout sista'

File: staff/Melinx/css/styles/main.css [hunk 9]
```diff
@@ -0,0 +1,39 @@
+h1 {color:greenyellow}
```
Comment (agandia9) [inline staff/Melinx/css/styles/main.css:1]: correct indentation..

Comment (manuelbarzi) [reply staff/Melinx/css/styles/main.css:1]: ADD ; at end of rule

**Pull request merged**